app/assets/images/payment_icons/arhaus.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" role="img" aria-labelledby="p-arhaus"><title id="p-arhaus">Arhaus</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M26.7 19.4c-.3 0-.6 0-.8-.1-.4-.1-1-.4-1.8-2.2-1.3-3-4.5-11.3-4.8-12-.3-.6-.3-.7-.5-.7-.1 0-.2.1-.5.8l-4.7 12.4c-.4 1-.8 1.8-1.7 1.9h-.6c-.1 0-.2 0-.2.1s.1.2.4.2c1 0 2.1-.1 2.3-.1s1.1.1 1.6.1c.2 0 .3 0 .3-.2 0-.1-.1-.1-.2-.1h-.2c-.4 0-.8-.2-.8-.6 0-.5.1-.9.3-1.3l1.2-3.5c0-.1.1-.2.2-.2h4.4c.1 0 .2 0 .2.1l1.9 4.9c.1.3 0 .5-.1.5s-.1.1-.1.1c0 .1.2.1.6.1h3.2c.2 0 .4 0 .4-.2.2 0 .1 0 0 0zm-6.2-6.3h-3.9c-.1 0-.1 0-.1-.1l1.8-5.4c.1-.3.2-.3.3 0l2 5.4s0 .1-.1.1z" fill="#1e372f"/></svg>
```

Comment:
CI is complaining about the `id` of the title. Hopefully this will be the last change.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 38 24" width="38" height="24" role="img" aria-labelledby="pi-arhaus"><title id="pi-arhaus">Arhaus</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" opacity=".07"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff"/><path d="M26.7 19.4c-.3 0-.6 0-.8-.1-.4-.1-1-.4-1.8-2.2-1.3-3-4.5-11.3-4.8-12-.3-.6-.3-.7-.5-.7-.1 0-.2.1-.5.8l-4.7 12.4c-.4 1-.8 1.8-1.7 1.9h-.6c-.1 0-.2 0-.2.1s.1.2.4.2c1 0 2.1-.1 2.3-.1s1.1.1 1.6.1c.2 0 .3 0 .3-.2 0-.1-.1-.1-.2-.1h-.2c-.4 0-.8-.2-.8-.6 0-.5.1-.9.3-1.3l1.2-3.5c0-.1.1-.2.2-.2h4.4c.1 0 .2 0 .2.1l1.9 4.9c.1.3 0 .5-.1.5s-.1.1-.1.1c0 .1.2.1.6.1h3.2c.2 0 .4 0 .4-.2.2 0 .1 0 0 0zm-6.2-6.3h-3.9c-.1 0-.1 0-.1-.1l1.8-5.4c.1-.3.2-.3.3 0l2 5.4s0 .1-.1.1z" fill="#1e372f"/></svg>
```